Ladybird/Android/src/main/java/org/serenityos/ladybird/LadybirdActivity.kt

```diff
@@ -28,16 +31,24 @@ class LadybirdActivity : AppCompatActivity() {
         binding = ActivityMainBinding.inflate(layoutInflater)
         setContentView(binding.root)
         setSupportActionBar(binding.toolbar)
+        urlEditText = binding.urlEditText
         view = binding.webView
+        view.onLoadStart = { url: String, _ ->
+            urlEditText.setText(url, TextView.BufferType.EDITABLE)
+        }
+        urlEditText.setOnEditorActionListener { textView: TextView?, i: Int, _: KeyEvent? ->
```

Comment:
```suggestion
        urlEditText.setOnEditorActionListener { textView: TextView, actionId: Int, _: KeyEvent?
```
Or does that not work?

Reply:
I can get rid of the `Boolean` return type and  let it infer it at least.